staging/src/k8s.io/api/discovery/v1alpha1/types.go

```diff
@@ -57,6 +57,8 @@ type AddressType string
 const (
 	// AddressTypeIP represents an IP Address.
 	AddressTypeIP = AddressType("IP")
```

Comment:
Does IP mean just IPv4 or both that an v6? If just v4, maybe you should add a v6? If it means both, then the documentation [below](https://github.com/kubernetes/kubernetes/pull/84091/files#diff-d12d4e96fc00e14e7c2bd68e23fd2284R67-R68) is confusing and should be explained:

> The contents of this field are interpreted according to the corresponding EndpointSlice addressType field. This allows for cases like dual-stack (IPv4 and IPv6) networking.

Reply:
This is currently meant to include both v4 and v6 addresses. When created by the EndpointSlice controller, this mirrors the value of `PodIPs` on the corresponding Pod (also containing v4 and v6 addresses). I'll update the comment to clarify that. It's also worth noting that there is some ongoing discussion around the merits of separating this into separate attributes, so there's a chance this could change before the next release.